migration/migration_blackbox_test.go

```diff
@@ -744,6 +744,9 @@ func testMigration86(t *testing.T) {
 }
 
 func testMigration87SpaceTemplates(t *testing.T) {
+	migrateToVersion(t, sqlDB, migrations[:87], 87)
+	// Import test space with work items and see if migration to space templates works
+	assert.Nil(t, runSQLscript(sqlDB, "088-space-templates.sql"))
```

Comment:
is it `87` or `88` ? there's a change in `migration/migration.go` for the step 87.

Reply:
@xcoulon oh, I see. Let me address that.

Reply:
Fixed in 176a61b6